parsons/newmode/newmode.py

```diff
@@ -0,0 +1,182 @@
+from Newmode import Client
+from parsons.utilities import check_env
+from parsons.etl import Table
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class Newmode:
+    """
+    Args:
+        api_user: str
+            The Newmode api user. Not required if ``NEWMODE_API_USER`` env variable is
+            passed.
+        api_password: str
+            The Newmode api password. Not required if ``NEWMODE_API_PASSWORD`` env variable is
+            passed.
+        api_version: str
+            The Newmode api version. Defaults to "v1.0" or the value of ``NEWMODE_API_VERSION``
+            env variable.
+    Returns:
+        Newmode class
+    """
+
+    def __init__(self, api_user=None, api_password=None, api_version=None):
+
+        self.api_user = check_env.check('NEWMODE_API_USER', api_user)
+        self.api_password = check_env.check('NEWMODE_API_PASSWORD', api_password)
+
+        if (api_version is None):
+            api_version = "v1.0"
+
+        self.api_version = check_env.check('NEWMODE_API_VERSION', api_version)
+
+        self.client = Client(api_user, api_password, api_version)
+
+    def convertToTable(self, data):
+        # Internal method to create a Parsons table from a data element.
+        table = None
+        if (type(data) is list):
+            table = Table(data)
+        else:
+            table = Table([data])
+
+        return table
+
+    def getTools(self, params={}):
```

Comment:
In the name of ensuring we have complete documentation, we like to make sure all of our Connectors' "public" methods have doc-strings that describe the method API. For an example, you can see the `VAN` connector's `find_person` method: https://github.com/move-coop/parsons/blob/73adc2504a7ca2d3edd91378676ede56e82324a6/parsons/ngpvan/people.py#L15-L47